packages/papirus-icon-theme/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/PapirusDevelopmentTeam/papirus-icon-theme
+TERMUX_PKG_DESCRIPTION="Papirus open source icon theme for Linux
+ Papirus is a SVG-based icon theme, drawing inspiration from Material Design
+ and flat design.
+ .
+ This package contains the following icon themes:
+  - Papirus
+  - Papirus-Dark
+  - Papirus-Light
+  - ePapirus"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=20210601
+TERMUX_PKG_SRCURL=https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SHA256=e39f4212e7433010b6246f5a2989a9e263c7ab8029a5d1759f2744f4a54b6b14
+TERMUX_PKG_DEPENDS="hicolor-icon-theme"
```

Comment:
You need to add relevant commands to execute below under the function termux_step_pre_configure or termux_step_configure and other functions... Please check the Contributor's wiki for relevant information

Reply:
I do not now what to do and much less with this message from the Makefile:

```
make: Nothing to be done for 'all'.
```

Reply:
> make: Nothing to be done for 'all'.

Does package uses a build system supporting out-of-tree builds? If no \-\-\> use `TERMUX_PKG_BUILD_IN_SRC=true`.

Reply:
Ok

Reply:
It seems that it is already solved : 

```
/build-package.sh papirus-icon-themeBuilding dependency libc++ if necessary...
libc++@21d-1 installed - skipping
Building dependency hicolor-icon-theme if necessary...
hicolor-icon-theme@0.17-26 installed - skipping
termux - building papirus-icon-theme for arch arm...
Downloading https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/refs/tags/20210601.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:100   161  100   161    0     0    347      0 --:--:-- --:--:-- --:--:--   348
100  122k    0  122k    0     0  99197      0 --:--:--  0:00:01 --:--:100 3685k    0 3685k    0     0  1625k      0 --:--:--  0:00:02 --:--:100 7455k    0 7455k    0     0  2281k      0 --:--:--  0:00:03 --:--:100 10.3M    0 10.3M    0     0  2360k      0 --:--:--  0:00:04 --:--:100 14.6M    0 14.6M    0     0  2845k      0 --:--:--  0:00:05 --:--:100 18.3M    0 18.3M    0     0  2993k      0 --:--:--  0:00:06 --:--:100 19.1M    0 19.1M    0     0  2470k      0 --:--:--  0:00:07 --:--:100 21.9M    0 21.9M    0     0  2716k      0 --:--:--  0:00:08 --:--:100 25.6M    0 25.6M    0     0  2830k      0 --:--:--  0:00:09 --:--:100 26.5M    0 26.5M    0     0  2840k      0 --:--:--  0:00:09 --:--:-- 2834k
make: Nothing to be done for 'all'.
mkdir -p /data/data/com.termux/files/usr/share/icons
cp -R ./Papirus-Dark ./Papirus-Light ./Papirus ./ePapirus /data/data/com.termux/files/usr/share/icons
```

Reply:
Pd: It's an icon issue, I forgot to specify that the architecture was all